submission/persistence-mongo/src/test/kotlin/ac/uk/ebi/biostd/persistence/doc/service/SubmissionMongoPersistenceServiceTest.kt

```diff
@@ -1,94 +0,0 @@
-package ac.uk.ebi.biostd.persistence.doc.service
-
-import ac.uk.ebi.biostd.persistence.common.request.SubmissionRequest
-import ac.uk.ebi.biostd.persistence.doc.db.data.SubmissionDocDataRepository
-import ac.uk.ebi.biostd.persistence.doc.db.data.SubmissionRequestDocDataRepository
-import ac.uk.ebi.biostd.persistence.doc.model.SubmissionRequestStatus.PROCESSED
-import ac.uk.ebi.biostd.persistence.filesystem.request.FilePersistenceRequest
-import ac.uk.ebi.biostd.persistence.filesystem.service.FileSystemService
-import ebi.ac.uk.extended.model.FileMode.COPY
-import io.github.glytching.junit.extension.folder.TemporaryFolderExtension
-import io.mockk.every
-import io.mockk.impl.annotations.MockK
-import io.mockk.junit5.MockKExtension
-import io.mockk.verify
-import org.assertj.core.api.Assertions.assertThat
-import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.extension.ExtendWith
-import uk.ac.ebi.extended.serialization.service.ExtSerializationService
-import uk.ac.ebi.extended.test.SubmissionFactory.defaultSubmission
-
-@ExtendWith(MockKExtension::class, TemporaryFolderExtension::class)
-class SubmissionMongoPersistenceServiceTest(
```

Comment:
Will you add new tests here as well?

Reply:
yes will add integration test now that this class does not have logic